### ISSUE TITLE
ask for pretrained hashing retrieval model

### ISSUE THREAD

**Issue by Raineasy** [caoyuan57/TUA#1]
Hello dear author
What a nice work！ It is glad to see this paper in the ICMR2024. But i also meet some difficulty when using the code to train the hashing model like DPSH and HashNet. The MAP metric always so low that i can‘t go on the experiment, could u share the pre-trained model checkpoint like DPSH HashNet and the model ResNet18 and vgg?

best wishes again!

**Comment by caoyuan57** [comment]
You can follow the link: "https://github.com/swuxyj/DeepHash-pytorch" to realize DSPH and HashNet, the pre-trained models ResNet18 and vgg  will be automatically downloaded.
